### DISCUSSION TITLE
Carthage TestsPerformance-iOS Fails

### DISCUSSION THREAD

**Discussion by AndreParenteNeon** [krzyzanowskim/CryptoSwift#1054] [Q&A]
![image](https://github.com/user-attachments/assets/0f3881fe-5f62-41fe-b22f-cd088ffc1c26)
trying to run `carthage update --platform ios --use-xcframeworks --no-use-binaries` but got stuck in this error.. anyone could help? trying to update to latest version 1.8.3 (from current 1.8.1 version) -> trying to update xcode 15.3 to xcode 16

**Comment by krzyzanowskim** [comment]
pperformance tests are not meant to run casually (due to many reasons). Let me check why it runs at all for carthage

**Reply by AndreParenteNeon** [reply]
thanks @krzyzanowskim ! I could investigate this either... if you need help for testing feel free to reach me
